packages/driver/src/cy/logGroup.ts

```diff
@@ -0,0 +1,42 @@
+import { $Command } from '../cypress/command'
```

Comment:
Am I missing something or is this file not used anywhere?

Reply:
You aren't missing anything. It's not used anywhere in this PR. The follow up work will be to:
1. use in `cy.switchToDomain`
2. use in `cy.sessions`
3. use in `.within()`

Reply:
I'd typically agree I'd prefer to see code be consumed when it is introduced, however this is intended to be an incremental change. Plus, the follow up work requires changes in the multi-domain feature branch and in the 10.0-release branch so it's best to keep develop as the source of truth rather than juggle coordinating changes in both branches.